Source/UI/Controllers/AddressViewController.swift

```diff
@@ -55,10 +55,10 @@ public class AddressViewController: UIViewController,
         countrySelectionViewController.delegate = self
         addTextFieldsDelegate()
 
-        if let regionCodeSelectedUnwrap = regionCodeSelected {
-            let country = Country(iso3166Alpha2: regionCodeSelectedUnwrap, dialingCode: nil)
+        if let country = regionCodeSelected.flatMap({ Country.from(iso3166Alpha2: $0) }) {
```

Comment:
why using `flatMap` for string for ? `Country.from(iso3166Alpha2`
it can be 
```suggestion
if let regionCodeSelectedUnwrap = regionCodeSelected {
let country = Country.from(iso3166Alpha2: regionCodeSelectedUnwrap)
```